.gitlab-ci.yml

```diff
@@ -33,9 +35,65 @@ build-composer-dependencies:
 
 phpcs:
   stage: code_quality
+  needs:
+    - build-composer-dependencies
   script:
     - composer ci:php:sniff
 
+php-lint-php7.2:
+  image: php:7.2
+  stage: lint
+  needs:
+    - build-composer-dependencies
```

Comment:
The PHP linting does not need the Composer dependencies. (Also for the other PHP versions.)